### PR TITLE
add .direnv to default exclude argument

### PR DESCRIPTION
**Pull request by fpgmaas**
**PR Checklist**

-   [x] A description of the changes is added to the description of this PR.
-   [x] If there is a related issue, make sure it is linked to this PR.
-   [ ] If you've fixed a bug or added code that should be tested, add tests!
-   [x] Documentation in `docs` is updated

**Description of changes**

### CODE REVIEW

File: poetry.lock [hunk 1]
```diff
@@ -10,7 +10,7 @@ python-versions = ">=3.5"
 dev = ["cloudpickle", "coverage[toml] (>=5.0.2)", "furo", "hypothesis", "mypy (>=0.900,!=0.940)", "pre-commit", "pympler", "pytest (>=4.3.0)", "pytest-mypy-plugins", "sphinx", "sphinx-notfound-page", "zope.interface"]
 docs = ["furo", "sphinx", "sphinx-notfound-page", "zope.interface"]
 tests = ["cloudpickle", "coverage[toml] (>=5.0.2)", "hypothesis", "mypy (>=0.900,!=0.940)", "pympler", "pytest (>=4.3.0)", "pytest-mypy-plugins", "zope.interface"]
-tests-no-zope = ["cloudpickle", "coverage[toml] (>=5.0.2)", "hypothesis", "mypy (>=0.900,!=0.940)", "pympler", "pytest (>=4.3.0)", "pytest-mypy-plugins"]
```
Comment (mkniewallner) [inline poetry.lock:13]: Since those changes are not related to the PR, should they be included in the PR? Also, you might want to update Poetry to its latest version ([1.2.2](https://github.com/python-poetry/poetry/releases/tag/1.2.2)), as extras are now normalised, to avoid a back-and-forth in the names (especially since Renovate always uses latest version of Poetry).

**Comment by mkniewallner** [comment]
The default list of excludes makes me think that we could benefit from ignoring all files that are in `.gitignore`, similarly to what [black does](https://black.readthedocs.io/en/stable/usage_and_configuration/file_collection_and_discovery.html#gitignore).

If you think that it makes sense, I'd be happy to create an issue for it.

File: poetry.lock (hunk 1 above)
Comment (fpgmaas) [reply poetry.lock]: @mkniewallner Thanks, missed this! I reset poetry.lock to the version in `main`.

File: docs/usage.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ _deptry_ can also be configured to look for `requirements.txt` files with other
 
 To determine issues with imported modules and dependencies, _deptry_ will scan the working directory and its subdirectories recursively for `.py` and `.ipynb` files, so it can
 extract the imported modules from those files. Any files solely used for development purposes, such as files used for unit testing, should not be scanned. By default, the directories
-`venv`, `.venv`, `tests`,`.git` and the file `setup.py` are excluded.
+`venv`, `.venv`, `.direnv`, `tests`,`.git` and the file `setup.py` are excluded.
```
Comment (mkniewallner) [inline docs/usage.md:41]:
```suggestion
`venv`, `.venv`, `.direnv`, `tests`, `.git` and the file `setup.py` are excluded.
```

Comment (fpgmaas) [reply docs/usage.md]: Thanks just fixed it because of the merge conflict :)

**Review by mkniewallner — APPROVED** [review]

**Comment by fpgmaas** [comment]
> The default list of excludes makes me think that we could benefit from ignoring all files that are in `.gitignore`, similarly to what [black does](https://black.readthedocs.io/en/stable/usage_and_configuration/file_collection_and_discovery.html#gitignore).
> 
> If you think that it makes sense, I'd be happy to create an issue for it.

@mkniewallner That definitely makes sense :) Would have also prevented the related issue from popping up indeed.
